lib/controllers/v1/taxa_controller.js

```diff
@@ -696,9 +690,7 @@ TaxaController.searchQuery = async ( req, opts = { } ) => {
       source: { _source: req._source || defaultSource }
     } );
   } else {
-    data = await esClient.connection.search( {
-      preference: global.config.elasticsearch.preference,
-      index: `${process.env.NODE_ENV || global.config.environment}_taxa`,
+    data = await esClient.search( "taxa", {
       body: elasticQuery,
       _source: req._source || defaultSource
```

Comment:
Here we can just add `_source` to the `elasticQuery` object, e.g. adding `elasticQuery._source = req._source || defaultSource;` just before calling `esClient.search`